includes/base/controls-stack.php

```diff
@@ -2024,6 +2024,10 @@ public function print_render_attribute_string( $element ) {
 		echo $this->get_render_attribute_string( $element ); // XSS ok.
 	}
 
+	protected function single_content_template() {
+		return false;
+	}
```

Comment:
Same here. Move to `Widget_Nested_Base` and change to `protected function single_content_template() {}` - empty method.